deb/Makefile

```diff
@@ -3,7 +3,8 @@ include ../common.mk
 CLI_DIR=$(realpath $(CURDIR)/../../cli)
 ENGINE_DIR=$(realpath $(CURDIR)/../../engine)
 PLUGINS_DIR=$(realpath $(CURDIR)/../plugins)
-GITCOMMIT?=$(shell cd $(CLI_DIR) && git rev-parse --short HEAD)
+CLI_GITCOMMIT?=$(shell cd $(CLI_DIR) && git rev-parse --short HEAD)
+ENGINE_GITCOMMIT?=$(shell cd $(ENGINE_DIR) && git rev-parse --short HEAD)
```

Comment:
What we should do (to be sure we don't break anything), is

- if `CLI_GITCOMMIT` is not set, but `GITCOMMIT` is, then `CLI_GITCOMMIT=GITCOMMIT`
- same for `ENGINE_GITCOMMIT`

Reply:
@thaJeztah are you sure for the `ENGINE_COMMIT`, shouldn't we do the `git rev-parse` from the engine directory to be sure we use the correct commit and not the default `GITCOMMIT` hash which can come from the `CLI_DIR`?

Reply:
Updated the `ENGINE_GITCOMMIT` too